src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -85,6 +86,51 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
       case 'health_care_region_detail_page':
         addGetUpdatesFields(pageCompiled, pages);
         break;
+      case 'event_listing':
+        modListPages(
+          pageCompiled,
+          files,
+          page.allEventTeasers,
+          'event_listing.drupal.liquid',
+          'event',
+        );
+        break;
```

Comment:
What about this pattern:

```suggestion
        pageCompiled.allEventTeasers = pageCompiled.fieldOffice.entity
          .reverseFieldOfficeNode.entities.length
          ? pageCompiled.fieldOffice.entity.reverseFieldOfficeNode
          : pageCompiled.reverseFieldOfficeNode;
        addPager(
          pageCompiled,
          files,
          page.allEventTeasers,
          'event_listing.drupal.liquid',
          'event',
        );
```

Where `addPages` is the bottom part of `modListPages` stripped out:

```js

function addPager(page, files, field, template, aria) {
  // Add our pager to the Drupal page.
 //  if (page.entityBundle !== 'health_services_listing') {
    const pagingObject = paginatePages(page, files, field, template, aria);
    if (pagingObject[0]) {
      page.pagedItems = pagingObject[0].pagedItems;
      page.paginator = pagingObject[0].paginator;
    }
  //}
}
```

Reply:
Yeah, that's pretty slick :+1: - will implement